test/unit/extract/xbrl_test.py

```diff
@@ -154,7 +158,17 @@ def test_ferc_xbrl_datastore_get_filings(mocker, file_map, selected_filings):
             ),
             [form for form in XbrlFormNumber if form != XbrlFormNumber.FORM1],
         ),
-        (FercToSqliteSettings(), []),
+        (
+            FercToSqliteSettings(
```

Comment:
`FercToSqliteSettings` will now create default settings for each form if they are all `None`.